doc/RELEASING.md

```diff
@@ -1,19 +1,45 @@
 
-## Steps to relase
 The release manager must do two things on every release day:
 
 ### Prepare next release
-1. At the end of the working day (EOD), a branch is cut from `master` named in a specific format. The format has two parts, an identifier that it is a **release** branch, and the date on which the branch is scheduled to go live. For example, the release branch cut on November 7th 2018 will be named `release/2018-11-21` to indicate that it is the release which is scheduled to go live on 21st November 2018.
-2. A new section is added to the [`CHANGELOG`](https://github.com/simpledotorg/simple-android/wiki/Changelog) which will contain the release notes for the upcoming release branch. This will serve as a reference to QA for testing and for release notes when the update is live.
-3. This branch will remain in QA for **one week** until release day. During this period, QA can build this branch from CI (process to build this branch will be described later) and can verify the builds.
-4. Certain commits which get merged into `master` when this branch is in QA will get cherry-picked into this branch. Only a few categories of commits are eligible for being cherry-picked, examples being bug fixes and copy changes. Whenever this happens, the release section in the changelog will have to be updated. 
-5. Whenever a commit is cherry-picked from `master` onto the release branch, it needs to be done with the `-x` flag to record the SHA1 hash of the commit on `master`. 
 
-### Complete current release
+1. At the end of the working day, a branch is cut from `master` named in a specific format. 
+   - The format has two parts, an identifier that it is a _release_ branch, and the date on which the branch is scheduled to go live. 
+   - For example, the release branch cut on November 7th 2018 will be named `release/2018-11-21` to indicate that it is the release which is scheduled to go live on 21st November 2018.
+    
+1. Once the release branch has been made, the `demo` variant is built from this branch and released to [Simple Demo](https://play.google.com/store/apps/details?id=org.simple.clinic.staging) on the Play Store.
+   - There is a workflow on Bitrise to do this.
+   - The Demo app is open to all. Anyone may download and test new features that will soon be available on production.
+   - No data on the Demo app is real patient data, everything is fake. People are encouraged to add their own data and try things out.
+  
+### Fixing issues
+
+  1. If someone discovers a problem in the upcoming release, it must be fixed and merged into `master`, and then cherry-picked into the release branch.
+  1. Only a few categories of issues are eligible for cherry-picking: bug fixes and language/text changes. All other types of issues will have to wait till the next release goes out. 
+  1. Whenever a commit is cherry-picked from `master` onto the release branch, it should be done with the `-x` flag to record the SHA1 hash of the original commit on `master`. 
```

Comment:
Why… is every point labeled with `1.`? :o

Reply:
Because markdown does automatic numbering when rendered...?

Reply:
I thought about this again, and even though markdown does automatic numbering, while reading the file itself -- especially in an editor -- it will look really silly to see a series of consecutive 1s.

Will fix this ASAP.

Reply:
True